Antidote/ContactCellView.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  ContactCellView.swift
+//  Antidote
+//
+//  Created by Yoba on 18/12/2016.
+//  Copyright © 2016 Dmytro Vorobiov. All rights reserved.
+//
+
+import Cocoa
+import SnapKit
+
+/**
+    ContactCellView is used in the ContactsTableViewController and represents a cell with short information about user's contact (or friend): contact's username, last message, avatar.
+ 
+    This is also used for few actions: calling a contact, checking it's full information, call history, etc.
+ */
+
+// MARK: init methods and properties
+
+class ContactCellView: NSTableCellView {
+    var userAvatar: RoundedImageView!
+    var lastSenderAvatar: RoundedImageView!
```

Comment:
User is your friend/interlocutor, there is always his/her/its avatar. 
The last sender is the person who sent last message in conversation. It may be you, your friend or others people in group chat. 
I don't think this feature is required, but until alpha-version let it be.